src/building.md

```diff
@@ -239,3 +239,99 @@ git clone https://github.com/spack/spack.git
 cd spack
 ./bin/spack install -v mfem
 ```
+
+## Building MFEM with CMake
+To build a serial form of MFEM with CMake first create a build directory. For example, using a build directory named `build`:
+```sh 
+mkdir build 
+cd build 
+```
+
+Run the CMake configuration on the MFEM source directory.
+```sh 
+cmake ..
+```
+
+Run the build command associated with the CMake configuration, specifying the number of parallel build tasks with the `-j` flag (4 tasks in this case). 
+```sh 
+cmake --build . -j 4
+```
+### Parallel build using CMake
+To build a parallel form of MFEM with CMake first build METIS and Hypre as described above.
+From the MFEM source directory, create a build directory. For example, using a build directory named `build`:
+```sh 
+cd mfem-4.5
```

Comment:
don't add versions to general instructions

Reply:
This was something I had a question about while writing this, the Makefile section of these instructions assume `mfem-4.5`, should I stick to that convention or use "MFEM_DIR", and should the Makefile instructions do the same?